examples/config-boltdb.json

```diff
@@ -0,0 +1,20 @@
+{
+    "distSpecVersion": "1.0.1-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot",
+        "dedupe": true,
+        "cache": {
+            "type": "boltdb",
+            "cacheName": "cache1",
+            "rootDirectory": "/tmp/zot/cachedb",
```

Comment:
Why do we have another rootDirectory param here?

Reply:
Removed